core/transaction.go

```diff
@@ -200,7 +204,7 @@ func TransactionDiff() {
 	}
 
 	spinner, _ := cmdr.Spinner.Start("Gathering changes made by transaction...")
-	cmd := exec.Command("diff", "-qr", "/.system", "/partFuture")
+	cmd := exec.Command("rsync", "-avxHAXSn", "--delete", "--out-format=%i%n", "/.system", "/partFuture")
```

Comment:
Shouldn't we be doing a dry-run to make no changes? Or does one of the flags here take care of this already?

Reply:
`-n` is short for `--dry-run`

Reply:
oh cool!